src/components/registration/Registration.tsx

```diff
@@ -195,7 +195,17 @@ export const Registration: React.FunctionComponent<RegistrationProps> = ({
                 placeholder="Phone #"
                 aria-label="Phone #"
                 onChange={(event: React.ChangeEvent<{ value: unknown }>) => {
-                  handleOnChange(event)
+                  const { value } = event.target as any
+                  if (value.includes('+')) {
```

Comment:
in the possible scenario that value is null,
value.includes(+) will fail , saying that can not apply includes from "null", 

can we check for null before this line? or 
ensure value is thruty before calling includes in this line? 
(anything thas more clean to read)